src/Controller/RefileRequestController.php

```diff
@@ -111,19 +111,27 @@ public function createRefileRequest()
 
             // Track status issues for the database for NYPL only.
             $statusFlag = true;
+            $afMessage = null;
+            $sip2Response = null;
 
             APILogger::addDebug('Received SIP2 message', $result);
 
             // Log a failed SIP2 status change to AVAILABLE without terminating the request prematurely.
             if ($result['fixed']['Alert'] == 'Y') {
                 APILogger::addError('Failed to change status to AVAILABLE.' . ' (itemBarcode: ' . $refileRequest->getItemBarcode() . ')');
                 $statusFlag = false;
+                $afMessage = $result['variable']['AF'];
+                $sip2Response = json_encode($result);
             }
 
             $refileRequest->addFilter(new Filter('id', $refileRequest->getId()));
             $refileRequest->read();
             $refileRequest->update(
-                ['success' => $statusFlag]
+                [
+                    'success' => $statusFlag,
+                    'af_message' => $statusFlag === false ? $afMessage :  null,
```

Comment:
As you're setting these variables to `null` already there's no need to do the `$statusFlag === false ?` bit. You can just assign the variable.